dev/make-distribution.sh

```diff
@@ -163,7 +163,7 @@ echo -e "\$ ${BUILD_COMMAND[@]}\n"
 rm -rf "$DISTDIR"
 mkdir -p "$DISTDIR/jars"
 echo "Spark $VERSION$GITREVSTRING built for Hadoop $SPARK_HADOOP_VERSION" > "$DISTDIR/RELEASE"
-echo "Build flags: $@" >> "$DISTDIR/RELEASE"
+echo "Build flags: "$@"" >> "$DISTDIR/RELEASE"
```

Comment:
@liu-zhaokun you're actually unquoting this argument as far as I can tell. It was fine before.